scss/components/_visibility.scss

```diff
@@ -70,6 +70,10 @@
     visibility: hidden;
   }
 
+  .visible {
+    visibility: visible !important;
```

Comment:
Why `!important`?

Reply:
Good call, tested and no need for `!important` here.